lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/ConfigurationService.java

```diff
@@ -72,22 +78,34 @@ public ConfigurationService(
         }
 
         // ****************************Private Parameters****************************
-
         this.parameterPrefix = System.getenv("AWS_STACK_NAME");
         this.thirdPartyId = paramProvider.get(String.format(KEY_FORMAT, env, "thirdPartyId"));
         this.contraindicationMappings =
                 paramProvider.get(getParameterName("contraindicationMappings"));
         this.dcsEndpointUri = paramProvider.get(getParameterName("dcsEndpoint"));
-        this.documentCheckResultTableName = paramProvider.get(getParameterName("FraudTableName"));
+        this.documentCheckResultTableName =
+                paramProvider.get(getParameterName("documentCheckResultTableName"));
 
         this.dcsSigningCert = getCertificate(paramProvider, "signingCertForDrivingPermitToVerify");
         this.dcsEncryptionCert =
-                getCertificate(paramProvider, "encryptionCertForPassportToEncrypt");
+                getCertificate(paramProvider, "encryptionCertForDrivingPermitToEncrypt");
+
+        // PASSPORT_CRI_TLS_CERT("/%s/credentialIssuers/ukPassport/self/tlsCert"),
+        this.drivingPermitTlsSelfCert = getCertificate(paramProvider, "tlsCert");
+
+        // DCS_TLS_ROOT_CERT("/%s/credentialIssuers/ukPassport/dcs/tlsRootCertificate"),
+        this.dcsTlsRootCert = getCertificate(paramProvider, "tlsRootCertificate");
```

Comment:
So do we have certs for driving licence or are we just re-using passports for the purposes of seeing what the response from DCS looks like?

Reply:
Reusing passport for now the best DCS responses are in that doc I sent. I dont have anything specific here